source/docs/software/pathplanning/system-identification/viewing-diagnostics.rst

```diff
@@ -62,44 +72,31 @@ One of the most-common errors is an inappropriate value for the motion threshold
 Motion Threshold Too Low
```

Comment:
It is called Velocity Threshold in the UI.  I think these should match, lets either change the UI or this.

Reply:
I prefer velocity threshold because it's more specific.